erpnext/hr/report/employee_leave_balance_summary/employee_leave_balance_summary.py

```diff
@@ -0,0 +1,96 @@
+# Copyright (c) 2013, Frappe Technologies Pvt. Ltd. and contributors
+# For license information, please see license.txt
+
+from __future__ import unicode_literals
+import frappe
+from frappe.utils import flt
+from frappe import _
+from erpnext.hr.doctype.leave_application.leave_application import get_leaves_for_period
+
+from ..employee_leave_balance.employee_leave_balance import get_total_allocated_leaves
```

Comment:
```suggestion
from erpnext.hr.employee_leave_balance.employee_leave_balance import get_total_allocated_leaves
```

Reply:
@Mangesh-Khairnar reason?

Reply:
Explicit is better than implicit

Reply:
@Mangesh-Khairnar Do you mean absolute import is better than relative import?

fixing it since we have not used such pattern elsewhere.